Items/07_Items.txt

```diff
@@ -43,6 +47,12 @@ from multiple angles to capture the whole object. Up to 20m^3 of space.
 Wearables
 ==============================
 
+Dog Harness ($45):
+    This harness comes with up to 18 characters of monogram and a micro
+chip that helps you find your dog if it runs away. The collar itself is 
```

Comment:
"harness itself"
Most of the time, collars and harnesses have ID tags, while the microchip is implanted directly into the animal (so they can be identified/found in the absence of any equipment).

Reply:
This doesn't preclude putting a chip in the harness as well.